fe/fe-core/src/main/java/com/starrocks/planner/FileScanNode.java

```diff
@@ -608,6 +615,11 @@ private void processFileGroup(
             TBrokerRangeDesc rangeDesc =
                     createBrokerRangeDesc(curFileOffset, fileStatus, formatType, rangeBytes, columnsFromPath,
                             numberOfColumnsFromFile);
+
+            rangeDesc.setStrip_outer_array(jsonOptions.stripOuterArray);
+            rangeDesc.setJsonpaths(jsonOptions.jsonPaths);
```

Comment:
if not set jsonPaths, it will be null?

Reply:
No. The `jsonOptions` is initialized as the default value in the class constructor.

Reply:
public static class JSONOptions {
        @SerializedName("s")
        public boolean stripOuterArray;

        @SerializedName("jp")
        public String jsonPaths;

        @SerializedName("jr")
        public String jsonRoot;
    }
  seems no initialized